Makefile

```diff
@@ -0,0 +1,270 @@
+SHELL:=/bin/bash
+UNAME:=$(shell uname)
+CURDIR_BASE:=$(shell basename "$(CURDIR)")
+export LOG_DIR_ABS:=$(shell python -c 'import os; print(os.path.realpath("logs"))')
+
+$(LOG_DIR_ABS):
+	mkdir -p "$(LOG_DIR_ABS)"
+
+define help
+This is the Makefile for setting up Ridgeback development instance
+
+Basic dev instance setup instructions:
+-------------------------------------
+
+1. install dependencies in the current directory with:
+make install
+
+2. initialize the database with:
+make db-init
+
+3. initialize the Django database and set a admin (superuser) account with:
+make django-init
+
+4. to run Riodgeback, first start Postgres, RabbitMQ, and Celery with:
+make start-services
+
+5. start the main Django development server with:
+make runserver
+
+Extras
+------
+
+Shutdown all services:
+make stop-services
+
+Check if pre-configured ports are already occupied on your system:
+make check-port-collision
+
+Consult the contents of this Makefile for other management recipes.
+
+endef
+export help
+help:
+	@printf "$$help"
+
+.PHONY : help
+
+
+# ~~~~~ Python and nextflow installation ~~~~~~ #
+PATH:=$(CURDIR)/conda/bin:$(PATH)
+unexport PYTHONPATH
+unexport PYTHONHOME
+
+ifeq ($(UNAME), Darwin)
+CONDASH:=Miniconda3-4.7.12.1-MacOSX-x86_64.sh
+endif
+
+ifeq ($(UNAME), Linux)
+CONDASH:=Miniconda3-4.7.12.1-Linux-x86_64.sh
+endif
+
+CONDAURL:=https://repo.continuum.io/miniconda/$(CONDASH)
+
+conda:
+	@echo ">>> Setting up conda..."
+	@wget "$(CONDAURL)" && \
+	bash "$(CONDASH)" -b -p conda && \
+	rm -f "$(CONDASH)"
+
+# NOTE: RabbitMQ sometimes has installation problems on macOS 10.12
+install: conda
+	conda install -y \
+	anaconda::postgresql \
+	conda-forge::ncurses \
+	rabbitmq-server && \
+	pip install -r requirements.txt
+
+# Ridgeback environment variables for configuration
+export RIDGEBACK_DB_NAME:=db
+export RIDGEBACK_DB_USERNAME:=$(shell whoami)
+export RIDGEBACK_DB_PASSWORD:=admin
+export RIDGEBACK_DB_URL:=localhost
```

Comment:
This works on my local machine (hitting a socket at `/var/run`) -- maybe due to another instance of postgres running. On Juno I had to update this `/tmp`.

Reply:
Which line is this referring to? Which directory did you try to execute from on Juno?

Reply:
@aef- I have it running on Silo with the current config here.

Reply:
I'm not sure why it defaults to searching for sockets in `/var/run` but just something to note in case anyone runs into an issue where django hangs